sympy/assumptions/ask.py

```diff
@@ -1481,7 +1517,7 @@ def get_known_facts():
         Equivalent(Q.extended_real, Q.real | Q.infinite),
         Equivalent(Q.even | Q.odd, Q.integer),
         Implies(Q.even, ~Q.odd),
-        Equivalent(Q.prime, Q.integer & Q.positive & ~Q.composite),
+        Implies(Q.prime, Q.integer & Q.positive & ~Q.composite),
```

Comment:
Why was this changed?

Reply:
Because `1` is neither prime nor composite. `satask` was giving the wrong answer for prime.

Reply:
Ah. I thought this was already well tested.

Reply:
Add a test for this.

Reply:
> Ah. I thought this was already well tested.

I figured it out because for sometimes I tested ask with `_eval_ask` functions performed after `satask`. This was the only error that popped.


> Add a test for this.

Sure